core-bundle/contao/widgets/SerpPreview.php

```diff
@@ -41,7 +41,7 @@ public function generate()
 		}
 
 		$id = $model->id;
-		$title = StringUtil::substr($this->getTitle($model), 64);
+		$title = StringUtil::restoreBasicEntities(StringUtil::substr($this->getTitle($model), 64));
```

Comment:
Shouldn‘t the basic entities be replaced before the title is shortened? At least the JavaScript behaves this way and I think it would make more sense, too.

Reply:
Absolutely right!

Also, it doesn't make sense to use `StringUtil::restoreBasicEntities()` in this case: We might end up with strings like `…Dampf&sh` if `substr` splits within an HTML entity.

Fixed in [e3002e0](https://github.com/contao/contao/pull/5566/commits/e3002e0ea90303c2ab24d50c43fd338c43861563)